src/lib.rs

```diff
@@ -26,8 +43,16 @@ pub struct Id<'id> {
 }
 
 impl<'id> Id<'id> {
-    // Do not use this function; use the `make_guard!` macro instead.
-    #[doc(hidden)]
+    /// Construct an `Id` with an unbound lifetime.
```

Comment:
unbound -> unbounded

See this [doc](https://doc.rust-lang.org/nomicon/unbounded-lifetimes.html)